apps/chat2/chat2.nim

```diff
@@ -34,11 +34,13 @@ import
   ../../waku/common/utils/nat,
   ./config_chat2
 
+
 when defined(rln) or defined(rlnzerokit):
   import
     libp2p/protocols/pubsub/rpc/messages,
     libp2p/protocols/pubsub/pubsub,
-    ../../waku/v2/protocol/waku_rln_relay/waku_rln_relay_utils
+    ../../waku/v2/protocol/waku_rln_relay/waku_rln_relay_utils,
+    ../../waku/v2/protocol/waku_rln_relay/rln_types
```

Comment:
We need to put all RLN modules behind a unique module. Please, check the other protocols (e.g., `protocols/waku_store.nim`, `protocols/waku_filter.nim`, etc.):

NB the "entry point" module is in the `protocols` folder. It imports the internal modules and reexports only those that should be accessible from outside).

With that approach, the internal modules can evolve without affecting/making the imports change in the app code (like here).